Chapter 06 - Surface, WebAPI and RenderMVC Controllers/02 - Surface Controllers.md

```diff
@@ -82,7 +82,10 @@ namespace MyNamespace
 
 `SurfaceController`'s are not meant to be secured for the backoffice and are always public.
 
-Surface controllers follow MVC convention for naming and routing.  Please reference official Umbraco and Microsoft documentation.
+##Special Routing##
+Surface controllers follow MVC convention for naming and routing, with one exception (see below). Please reference official Umbraco and Microsoft documentation.
+
+Surface controllers are a little different than normal controllers in that they will also work with any URL. That is because when you render a surface controller using the `BeginUmbracoForm` extension method, a hidden input field with a name of `ufprt` (short for "Umbraco Form Post RouTe") will be rendered. The value is an encrypted string that contains all the routing information, such as the controller, action method, and area. If you decrypt the value stored in the `ufprt` field generated by a form rendered with the action method in the above code example, the value would be `c=MyNameSurface&a=GetNames&ar=MyNameThing` (i.e., the controller, action, and area).
```

Comment:
@Nicholas-Westby, looks good with one nitpick perhaps. Rather that use `decrypt`, shouldn't it be `decode`? I'm assuming it's Base64 encoding instead of actual encryption (maybe I'm reading you too literally).